keras_cv/models/segmentation/deeplab_v3_plus/deeplab_v3_plus_test.py

```diff
@@ -74,6 +74,13 @@ def test_weights_change(self):
             self.assertNotAllEqual(w1, w2)
             self.assertFalse(ops.any(ops.isnan(w2)))
 
+    @pytest.mark.large
```

Comment:
Let's make this a forward-pass numerical verification as well (like [this test](https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/object_detection/yolo_v8/yolo_v8_detector_test.py#L238))

Reply:
done